rangeseekbar/src/main/java/org/florescu/android/rangeseekbar/RangeSeekBar.java

```diff
@@ -606,8 +606,7 @@ protected synchronized void onDraw(@NonNull Canvas canvas) {
         mRect.right = getWidth() - padding;
         canvas.drawRect(mRect, paint);
 
-        boolean selectedValuesAreDefault = (getSelectedMinValue().equals(getAbsoluteMinValue()) &&
-                getSelectedMaxValue().equals(getAbsoluteMaxValue()));
+        boolean selectedValuesAreDefault = (normalizedMinValue < 0.00001 && normalizedMaxValue > 0.99999);
```

Comment:
Not sure what you mean... just update it on your branch, commit and push to that branch. It will update the Pull Request.